CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and **Merged pull requests**. All closed issued and bug fixes should be
 represented by the pull requests that fixed them. This log originated with Singularity 2.4
 and changes prior to that are (unfortunately) done retrospectively. Critical items to know are:
 
+ - fixed * expansion during app runscript creation
```

Comment:
I think this shouldn't go here. This is intended to be a list of classes of changes to Singularity that warrant an entry into the changelog. This is not the place to list out changes!

Reply:
I respectfully disagree @bauerm97.  The last several releases have had bug fix sections in the changelog.  I believe this is a perfect candidate for that.